api_factory/templates/$namespace/$name_$version/$service/transports/http.py.j2

```diff
@@ -88,7 +88,12 @@ class {{ service.name }}HttpTransport({{ service.name }}Transport):
 
         # Send the request.
         response = self._session.post(
-            f'https://{self.SERVICE_ADDRESS}/$rpc/{{ '.'.join(method.meta.address.package) }}.{{ service.name }}/{{ method.name }}',
+            'https://{host}/$rpc/{package}.{service}/{method}'.format(
```

Comment:
Do we plan on making this interface similar to the grpc transport interface in which this transport also returns callables? I have a feeling this inconsistency will bite us in the future.

Reply:
This is a good point. There is no "HTTP stub" equivalent in this case, but returning a callable that can be called with the request argument has merit.

This is a significant change, so I will open an issue.

Reply:
Filed #7.